java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/GenericMqttListener.java

```diff
@@ -6,9 +6,9 @@
 import java.net.URI;
 
 /**
- * Implement this interface to handle events from the websocket server after {@link GenericMqttSubscriber#subscribe(GenericMqttMessageListener)}.
+ * Implement this interface to handle events from the websocket server after {@link GenericMqttSubscriber#subscribe(GenericMqttListener)}.
  */
-public interface GenericMqttMessageListener {
+public interface GenericMqttListener {
```

Comment:
I think we need to convey its MESSAGE listener I think

Reply:
hmm to so maybe we should remove onOpen/onClose methods and then we can call it MessageListener? as in current shape it is not only message listener :)

Reply:
javadoc should not mention about web socket implementation anyway :) and both handlers won't be available if we switch to pulsar client. What do you think?